src/test/java/uk/nhs/hee/tis/trainee/sync/service/TcsSyncServiceTest.java

```diff
@@ -195,8 +195,10 @@ void shouldPatchBasicDetailsWhenValidOperations(Operation operation) {
     service.syncRecord(person);
 
     TraineeDetailsDto expectedDto = new TraineeDetailsDto();
-    expectedDto.setTraineeTisId("idValue");
-    expectedDto.setPublicHealthNumber("publicHealthNumberValue");
+    if (!operation.equals(DELETE)) {
+      expectedDto.setTraineeTisId("idValue");
+      expectedDto.setPublicHealthNumber("publicHealthNumberValue");
+    }
```

Comment:
There is a separate sub-task for handling the Person delete scenario specifically which would take care of this, I can remove any changes to Person handling from this PR if you feel that's appropriate though.

Reply:
I just meant adding an assertion to the test rather than extending the implementation as part of this.
Not that it was verified before, but something like this:
```
...
      expectedDto.setPublicHealthNumber("publicHealthNumberValue");
      verifyNoMoreInteractions(personService);
    } else {
      verify(personService).deleteById("idValue");
    }
```